internal/models/user.go

```diff
@@ -933,6 +934,40 @@ func (u *User) FindOwnedFactorByID(tx *storage.Connection, factorID uuid.UUID) (
 	return &factor, nil
 }
 
+func (user *User) WebAuthnID() []byte {
+	if webAuthnID, ok := user.UserMetaData["web_authn_id"].(string); ok && webAuthnID != "" {
+		return []byte(webAuthnID)
+	}
```

Comment:
I'm not very keen to allow this from user meta data, because it means Malice could set this to whatever value they want and somehow wreak havoc.

Let's not. Also, this is not in the RFC is it? I haven't seen it I think.

Reply:
It's in the RFC but not called out - intention was to provide flexibility to the user but I guess there's no need to implement unless the user asks. Let's leave it out for now.